### DISCUSSION TITLE
Error: [object Object] is not a PostCSS plugin

### DISCUSSION THREAD

**Discussion by enes-get-local** [vercel/next.js#24697] [Help]
`next build` has started raise webpack error after upgraded next version to 10.2.0. I don't see any clue to figure it out. Anyone faced same issue ?

Versions: 
Next 10.0.2
React 17.0.1
next-pwa 5.2.7
node 12.18.3

`HookWebpackError: [object Object] is not a PostCSS plugin
`
![image](https://user-images.githubusercontent.com/70728456/116798393-9813a380-aaf7-11eb-8564-d06a20e13008.png)

next.config.js

```
const withPlugins = require('next-compose-plugins');
const withPWA = require('next-pwa');
const withImages = require('next-images');
const withFonts = require('next-fonts');

const ci_platform = process.env.CI_PLATFORM || 'web';

const compress = process.env.NEXT_COMPRESS ? !!process.env.NEXT_COMPRESS : true;
const assetPrefix = process.env.NEXT_PUBLIC_CDN_URL || '';
const buildId = process.env.NEXT_PUBLIC_BUILD_ID || null;

let configObj = {
  pwa: {
    // disable: process.env.NODE_ENV === 'development',
    dest: 'public',
  },
  // exportTrailingSlash: true,
  exportPathMap: function () {
    return {
      '/': { page: '/' },
    };
  },
  future: {
    webpack5: true,
  },
  reactStrictMode: true,
  poweredByHeader: false,
  compress,
  assetPrefix,
  generateBuildId: async () => buildId,
};

if (ci_platform === 'web') {
  configObj.i18n = {
    locales: ['en-US', 'es-US'],
    defaultLocale: 'en-US',
  };
}

module.exports = withPlugins([withImages, withFonts], withPWA({ ...configObj }));
```

**Comment by akellbl4** [comment]
Hi @enes-get-local,
Have you solved the problem? I assume the problem might be in the PostCSS config. Could you please provide your content of `postcss.config.js`?

**Reply by enes-get-local** [reply]
@akellbl4 i don't  even use postcss. Problem was related npm cache fixed by clean it. 
[https://github.com/vercel/next.js/issues/24696](https://github.com/vercel/next.js/issues/24696)
